custom-words.txt

```diff
@@ -829,23 +882,26 @@ francesouth
 freebsd
 freeflow
 frontdoor
+Frulian
```

Comment:
You're adding lots of words that are already in the list. Can you just revert this file and see if spellcheck still passes?

Reply:
I was getting merge confict, so I pulled main branch into mine, and all these words came up.

Reply:
Can you just revert this file and see if spellcheck still passes?